nixos/modules/programs/wayland/hyprland.nix

```diff
@@ -70,6 +70,7 @@ in
     (import ./wayland-session.nix {
```

Comment:
This kind of import is not needed in a module system? You can just enable these configs in another option, like `wayland.wayland-session.enable`?

Reply:
wayland session module is not meant to be used as an option and not in `modules-list`. So that won't work.